Integration/Support/ConfigSupport.php

```diff
@@ -4,11 +4,14 @@
 
 namespace MauticPlugin\HelloWorldBundle\Integration\Support;
 
+use Mautic\IntegrationsBundle\DTO\Note;
```

Comment:
Probably we should use more detailed variable name like IntegrationNote for DTO. We've already use Note entity, @escopecz what is your opinion?